### DISCUSSION TITLE
How to use Theme Editor and AppMerger?

### DISCUSSION THREAD

**Discussion by marcelo-estevam-nectar** [microsoft/powerapps-tools#727] [Q&A]
Hi, 

I've downloaded and extracted the zip files for Theme Editor and AppMerger, but every time that I try to open a .msapp file in any of theses .exe , the application crashes and closes.

Does these apps requires more steps or have some .Net Core SDK dependencies to install and work properly?

kind Regards

Marcelo Estevam

**Comment by schindld** [comment]
Pretty sure it's a bug due to PowerApps updating the themes (changed some names, removed some, added others), and the Theme Editor here just hasn't been updated to match. It chokes on the .msapp file and also when just importing the Themes.json file, if they were made recently in PowerApps.
